tests/test_acl_inner_src_mac_rewrite.py

```diff
@@ -0,0 +1,255 @@
+            dvs_acl.create_acl_table(TABLE_NAME, TABLE_TYPE, BIND_PORTS, stage="egress")
+            acl_table_id = dvs_acl.get_acl_table_ids(1)[0]
+            assert acl_table_id is not None
+            acl_table_group_ids = dvs_acl.get_acl_table_group_ids(len(BIND_PORTS))
+
+            dvs_acl.verify_acl_table_group_members(acl_table_id, acl_table_group_ids, 1)
+            dvs_acl.verify_acl_table_port_binding(acl_table_id, BIND_PORTS, 1, stage="egress")
+            #dvs_acl.verify_acl_table_action_list(acl_table_id, EXPECTED_ACTION_LIST)
+        finally:
+            dvs_acl.remove_acl_table(TABLE_NAME)
+            dvs_acl.remove_acl_table_type(TABLE_TYPE)
+            dvs_acl.verify_acl_table_count(0)
+
+    def test_InnerSrcMacRewriteAclRuleCreationDeletion(self, dvs, dvs_acl, innersrcmacrewrite_acl_table):
+
+        # This test checks for ACL rule creation(more than one) deletion for the table type inner src mac rewrite
+
+        self.setup_db(dvs)
+        # Add the rule
+        config_qualifiers = {"INNER_SRC_IP": "10.10.10.10/32", "TUNNEL_VNI": "5000"}
+        self.create_acl_rule(dvs, TABLE_NAME, RULE_NAME, config_qualifiers, priority="1000", action="60:BB:AA:C3:3E:AB")
+        dvs_acl.verify_acl_rule_status(TABLE_NAME, RULE_NAME, "Active")
+
+        # Verify the rule with SAI entries
+        members = dvs_acl.asic_db.wait_for_n_keys("ASIC_STATE:SAI_OBJECT_TYPE_ACL_ENTRY", 1)
+        for member in members:
+            fvs = dvs_acl.asic_db.wait_for_entry("ASIC_STATE:SAI_OBJECT_TYPE_ACL_ENTRY", member)
+            assert fvs.get("SAI_ACL_ENTRY_ATTR_FIELD_INNER_SRC_IP") == "10.10.10.10&mask:255.255.255.255"
+            assert fvs.get("SAI_ACL_ENTRY_ATTR_FIELD_TUNNEL_VNI") == "5000&mask:0xffffffff"
+            assert fvs.get("SAI_ACL_ENTRY_ATTR_PRIORITY") == "1000"
+            assert fvs.get("SAI_ACL_ENTRY_ATTR_ACTION_SET_INNER_SRC_MAC") == "60:BB:AA:C3:3E:AB"
+
+        # Add second rule
+        config_qualifiers = {"INNER_SRC_IP": "10.10.10.10/32", "TUNNEL_VNI": "5000"}
+        self.create_acl_rule(dvs, TABLE_NAME, RULE_NAME+"2", config_qualifiers, priority="9990", action="AB:BB:AA:C3:3E:AB")
+        dvs_acl.verify_acl_rule_status(TABLE_NAME, RULE_NAME, "Active")
+
+        # Verify the rule2 status in ASIC_DB
+        members = dvs_acl.asic_db.wait_for_n_keys("ASIC_STATE:SAI_OBJECT_TYPE_ACL_ENTRY", 2)
+
+        # Remove rule and check status in ASIC_DB
+        self.remove_acl_rule(dvs, TABLE_NAME, RULE_NAME)
+        dvs_acl.verify_acl_rule_status(TABLE_NAME, RULE_NAME, None)
+
+        # Remove second rule and check status in ASIC_DB
+        self.remove_acl_rule(dvs, TABLE_NAME, RULE_NAME+"2")
+        dvs_acl.verify_acl_rule_status(TABLE_NAME, RULE_NAME+"2", None)
+
+        dvs_acl.verify_no_acl_rules()
+
+    def test_InnerSrcMacRewriteAclRuleCounterUpdate(self, dvs, dvs_acl, innersrcmacrewrite_acl_table):
```

Comment:
In the SWSS code there isn't any support for acl counter update, so this UT is not testing anything. If you add asserts for whether a counter actually exists I believe one won't exist.